packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/package.json.template

```diff
@@ -116,6 +120,7 @@
     "@o3r/schematics": "<%= sdkCoreRange %>"
   },<% } %>
   "peerDependencies": {
+    "@ama-sdk/client-fetch": "<%= sdkCoreRange %>",
```

Comment:
Mandatory because of `api-mock.ts`.
Created #2117 (containing breaking change) to remove it.